taxonomies/countries.txt

```diff
@@ -3332,6 +3332,7 @@ country_code_3:en: AUT
 language_codes:en: de
 osm_relation:en: 16239
 wikidata:en: Q40
+ecobalyse_is_part_of_eu: 1
```

Comment:
for boolean, by convention in taxonomies we use properties like "vegan:en: yes"
can you replace "ecobalyse_is_part_of_eu: 1" by "is_part_of_eu:en: yes" ?

Unless Ecobalyse has specific countries it considers to be part of EU, let's make the property name is_part_of_eu:en:

Reply:
Alright i'll change this ! 

Indeed, Ecobalyse considers specific countries to be part of EU sometimes in order to get realistic LCA (as an example the UK is not anymore in the EU but for transport impact it's considered so) so i would recommand to keep 'ecobalyse_is_part_of_eu'.